canonicalwebteam/store_api/stores/charmstore.py

```diff
@@ -3,14 +3,50 @@
 import requests
 
 from canonicalwebteam.store_api.store import Store
+from canonicalwebteam.store_api.publisher import Publisher
 
-SNAPSTORE_API_URL = getenv("CHARMSTORE_API_URL", "https://api.snapcraft.io/")
+CHARMSTORE_API_URL = getenv("CHARMSTORE_API_URL", "https://api.snapcraft.io/")
+CHARMSTORE_PUBLISHER_API_URL = getenv(
+    "CHARMSTORE_API_URL", "https://api.staging.snapcraft.io/"
```

Comment:
should it be the opposite? For now we put the non staging one by default and the url as en env var?

Reply:
Yeah, I thought about it as well. Okay, we can add this value to .env in charmhub.